servers/rendering/rendering_device.cpp

```diff
@@ -1063,9 +1079,9 @@ RID RenderingDevice::texture_create(const TextureFormat &p_format, const Texture
 	set_resource_name(id, "RID:" + itos(id.get_id()));
 #endif
 
-	if (p_data.size()) {
+	if (data.size()) {
 		for (uint32_t i = 0; i < p_format.array_layers; i++) {
-			_texture_initialize(id, i, p_data[i]);
+			_texture_initialize(id, i, data[i]);
```

Comment:
Looks like I forgot to actually use the new argument:

```suggestion
			_texture_initialize(id, i, data[i], immediate_flush);
```